terraform/scripts/setup_ssh_gateway.sh

```
@@ -0,0 +1,10 @@
+#!/bin/bash
+USER=devops
+
+gcloud compute ssh "$USER"@bastion --ssh-flag=-t $@ << 'EOF'
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)